shared/src/main/scala/squants/motion/Torque.scala

```diff
@@ -0,0 +1,61 @@
+package squants.motion
+
+import squants.mass.{Kilograms, MomentOfInertia, Pounds}
+import squants.space.{Feet, Meters}
+import squants.{AbstractQuantityNumeric, Dimension, Energy, PrimaryUnit, Quantity, SiBaseUnit, UnitConverter, UnitOfMeasure}
+
+/**
+  *
+  * @author paxelord
+  * @since 0.6
+  *
+  * @param value Double
+  */
+final class Torque private (val value: Double, val unit: TorqueUnit)
+    extends Quantity[Torque] {
+
+  def dimension = Torque
+
+  def toNewtonMeters = to(NewtonMeters)
+  def toPoundFeet = to(PoundFeet)
+
+  def / (that: MomentOfInertia): AngularAcceleration = {
+    RadiansPerSecondSquared(toNewtonMeters / that.toKilogramsMetersSquared)
+  }
+}
+
+object Torque extends Dimension[Torque] {
+  private[motion] def apply[A](n: A, unit: TorqueUnit)(implicit num: Numeric[A]) = new Torque(num.toDouble(n), unit)
+  def apply = parse _
+  def name = "Torque"
+  def primaryUnit = NewtonMeters
+  def siUnit = NewtonMeters
+  def units = Set(NewtonMeters, PoundFeet)
+}
+
+trait TorqueUnit extends UnitOfMeasure[Torque] with UnitConverter {
+  def apply[A](n: A)(implicit num: Numeric[A]) = {
+    Torque(num.toDouble(n), this)
+  }
+}
+
+object NewtonMeters extends TorqueUnit with PrimaryUnit with  SiBaseUnit {
+  val symbol = Newtons.symbol + "‧" + Meters.symbol
```

Comment:
This is meant to be the times symbol. I looked though squants, and I think this is the first time we have a times symbol in a unit. It might be clearer to replace the current bullet `‧` with a splat `*`

In a future PR, it might be useful to have have a `Product[Unit]` to allow for easy extendability of units. For example, `MomentOfInertia` would extend `Product[Area]` (because `Area` is `Length` squared) and `Product[Mass]` But that's for a future PR.